### PR TITLE
feat(default presets): remove `default:onlyNpm` preset, improve descriptions

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Remove the old `onlyNpm` preset
- Improve `description` for some default presets

## Context

Drive-by PR, not closing any issue with this.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/default.ts [hunk 1]
```diff
@@ -362,7 +362,7 @@ export const presets: Record<string, Preset> = {
     },
   },
   onlyNpm: {
-    description: 'Renovate only npm dependencies.',
+    description: 'Only update npm dependencies.',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/default.ts:365]: The description implies that Renovate will only update npm dependencies, but the config disables `docker-compose`, `dockerfile` and `meteor`.

I must be missing something here. In any case, the current `description` seems misleading to me.

Maintainer input wanted. 😉 

> ```
>   onlyNpm: {
>     description: 'Only update npm dependencies.',
>     'docker-compose': {
>       enabled: false,
>     },
>     dockerfile: {
>       enabled: false,
>     },
>     meteor: {
>       enabled: false,
>     },
>   },
> ```

File: lib/config/presets/internal/default.ts [hunk 2]
```diff
@@ -497,7 +497,7 @@ export const presets: Record<string, Preset> = {
   },
   renovatePrefix: {
     branchPrefix: 'renovate/',
-    description: 'Prefix `renovate/` to all branch names.',
+    description: 'Add the prefix `renovate/` to all branch names.',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/default.ts:500]:
```suggestion
    description: 'Add the `renovate/` prefix to all branch names.',
```

File: lib/config/presets/internal/default.ts (hunk 1 above)
Comment (rarkins) [reply lib/config/presets/internal/default.ts:365]: Ha, this is a very old preset! I think maybe time to remove it seeing as it clearly doesn't work anymore.

Comment (HonkingGoose) [reply lib/config/presets/internal/default.ts:365]: @rarkins want me to remove the `onlyNpm` preset then?

Comment (rarkins) [reply lib/config/presets/internal/default.ts:365]: Yes, I think we have a list of removed presets which it should be added to, that way users don't get _errors_ if they still include it

Comment (HonkingGoose) [reply lib/config/presets/internal/default.ts:365]: Is this the list you're thiking of?

https://github.com/renovatebot/renovate/blob/d10dcff495c7de06f324e0d26e6add6ee08747d6/lib/config/presets/common.ts#L1-L31

File: lib/config/presets/common.ts [hunk 3]
```diff
@@ -23,6 +23,7 @@ export const removedPresets: Record<string, string | null> = {
   'default:app': 'config:js-app',
   'default:js-app': 'config:js-app',
   'default:library': 'config:js-lib',
+  'default:onlyNpm': null,
```
Comment (HonkingGoose) [inline lib/config/presets/common.ts:26]: Please double check if this is the correct preset name.

Is there anything else I need to do to properly remove/migrate the `onlyNpm` preset?

The Creating/editing Renovate preset docs do **not** explain how to **remove** a preset:

https://github.com/renovatebot/renovate/blob/main/docs/development/creating-editing-renovate-presets.md

**Review by rarkins — APPROVED** [review]

**Pull request merged**